src/main/java/eu/europeana/metis/sandbox/config/ServicesConfig.java

```diff
@@ -22,8 +23,10 @@ MetricsService metricsService(
       RecordRepository recordRepository,
       RecordLogRepository recordLogRepository,
       DatasetProblemPatternRepository problemPatternRepository,
-      MeterRegistry meterRegistry) {
-    metricsService = new MetricsServiceImpl(recordRepository, recordLogRepository, problemPatternRepository, meterRegistry);
+      MeterRegistry meterRegistry,
+      AmqpConfiguration amqpConfiguration) {
+    metricsService = new MetricsServiceImpl(recordRepository, recordLogRepository, problemPatternRepository,
+            meterRegistry, amqpConfiguration, amqpConfiguration.getAmqpAdmin());
```

Comment:
I see this object depends `ampqConfiguration` is it possible to remove this parameter amqpConfiguration.getAmqppAdmin and use it from inside the `MetricsServiceImpl`?

Reply:
I left it like that initially for testing purposes, I wanted to inject a mock of AmqpAdmin into `MetricServiceImpl`
But I figured a different way of doing the tests without that extra parameter in the constructor, I'll push the changes